src/mappings/Reputation.graphql

```diff
@@ -0,0 +1,19 @@
+type ReputationHolder @entity {
+    contract: ID!
+    address: ID!
+    balance: BigInt!
+}
+
+type ReputationMint @entity {
```

Comment:
1. Are we sure we need to keep the history of mint/burn and not just the balance? Do you see a use case for that?
2. What about `totalSupply`?

Reply:
`Reputation` doesn't have a `name` and `symbol`. we do need to add `totalSupply` though.